charts/victoria-metrics-k8s-stack/templates/rules/additionalVictoriaMetricsRules.yml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.additionalVictoriaMetricsMap}}
+apiVersion: v1
+kind: List
+metadata:
+  name: {{ include "victoria-metrics-k8s-stack.fullname" $ }}-additional-victoria-metrics-rules
+  namespace: {{ .Release.Namespace }}
+items:
+{{- range $VMRuleName, $VMRule := .Values.additionalVictoriaMetricsMap }}
+  - apiVersion: operator.victoriametrics.com/v1beta1
+    kind: VMRule
+    metadata:
+      name: {{ template "victoria-metrics-k8s-stack.name" $ }}-{{ $VMRuleName }}
+      namespace: {{ $.Release.Namespace }}
+      labels:
+        app: {{ template "victoria-metrics-k8s-stack.name" $ }}
+{{ include "victoria-metrics-k8s-stack.labels" $ | indent 8 }}
```

Comment:
WDYT about replacing `indent` with `nindent`? It will create new line and potentially solve possible issues with indent, common yaml problem.

Reply:
Sure, that's good idea.